### DISCUSSION TITLE
node build_tgfx -p web 编译报错

### DISCUSSION THREAD

**Discussion by wuyasong** [Tencent/libpag#1906] [Q&A] [answered]
执行 `node build_tgfx -p web` 后报错，
设备信息: Apple M1 Pro mac os 13.0.1
fatal: unable to access 'https://github.com/glennrp/libpng.git/': HTTP/2 stream 1 was not closed cleanly before end of the underlying stream
node:child_process:935
    throw err;
    ^

Error: Command failed: depsync
fatal: unable to access 'https://github.com/glennrp/libpng.git/': HTTP/2 stream 1 was not closed cleanly before end of the underlying stream

另外想问下tgfx是否有API使用文档，可以像canvaskit一样

**Comment by domchen** [comment, marked as answer]
你这个报错是网络问题，提示访问不了 github ，更换一个网络或者配置一下科学上网。tgfx 是一个纯 C++ 库，在web 平台只支持作为 wasm 使用，并不提供 JS API 绑定。tgfx的具体使用可以参考项目里的测试用例。
